include/services/OswServiceTaskWiFi.h

```diff
@@ -44,6 +44,8 @@ class OswServiceTaskWiFi : public OswServiceTask {
     const String& getStationPassword() const;
 
     ~OswServiceTaskWiFi() {};
+     // ** updated added NTP status method to return NTP status 
+    bool getNTPstatus(void);
```

Comment:
Just comment `// blah blah`.
And `bool getNTPstatus();`

Reply:
can you be more specific, do you request that the comment be removed?

Reply:
What is that `**`?  ..  😄

Reply:
And Just `bool getNTPstatus();` not `bool getNTPstatus(void);`.
For the other things and the rules.

Reply:
ok you want the "void" removed ..